semconv-incubating/src/main/java/io/opentelemetry/semconv/incubating/DbIncubatingAttributes.java

```diff
@@ -237,19 +237,19 @@ public final class DbIncubatingAttributes {
   public static final AttributeKey<String> DB_USER = stringKey("db.user");
 
   // Enum definitions
-  /** Values for {@link #DB_CASSANDRA_CONSISTENCY_LEVEL}. */
-  public static final class DbCassandraConsistencyLevelValues {
+  /** Values for {@link #DB_CASSANDRA_CONSISTENCYLEVEL}. */
+  public static final class DbCassandraConsistencylevelValues {
```

Comment:
😢

Reply:
I agree, the only way it can be tolerated is if we avoided multi_word_names as much as possible.